### PR TITLE
Syndicate temperature gun.

### PR DESCRIPTION
**Pull request by ghost**
Adds Syndicate temperature gun. Uplink item for 14 tc. Works in two modes: bake and freeze. In freeze mode each beam decreases temperature of target. Also in slow mode slows target. In bake mode each beam increases temperature of target. Also in bake mode if target temperature is high enough, then sets target on fire. Has completed sprites (beams, sprites of weapon and in hand sprites).

### CODE REVIEW

File: code/modules/projectiles/projectile/special.dm [hunk 1]
```diff
@@ -72,6 +72,43 @@
...
+			M.IgniteMob()
+		}
+	return 1
+
+/obj/item/projectile/stemp/shot
+	name = "heat beam"
+	icon_state = "heat_1"
+	temperature = 400
+
+/obj/item/projectile/meteor
```
Comment (KazeEspada) [inline code/modules/projectiles/projectile/special.dm]: ???

**Comment by Bawhoppen** [comment]
:-1:x:100: Please don't add any more crappy nukeops weapons, its just boring and generic man

**Comment by KazeEspada** [comment]
We already have a temperature gun? And this doesnt seem to use any of its code?

Also, you did the cardinal git sin of making a pull request of master.

**Comment by HunterMaksim** [comment]
Ok

**Comment by KazeEspada** [comment]
Come join #coderbus on irc.

**Comment by HunterMaksim** [comment]
> Also, you did the cardinal git sin of making a pull request of master.
?

**Comment by Bawhoppen** [comment]
you didn't make a new branch on github for this

**Comment by HunterMaksim** [comment]
Also, this is weapon for both, traitors and nuke. It is really different from original temperature gun and because of what i haven't made it child of it.

**Pull request closed without merging**